msdsn/conn_str.go

```diff
@@ -72,6 +74,15 @@ type Config struct {
 	ConnTimeout time.Duration // Use context for timeouts.
 	KeepAlive   time.Duration // Leave at default.
 	PacketSize  uint16
+
+	// Kerberos authentication fields
+
+	Krb5ConfFile      string
+	KrbCache          string
```

Comment:
This line needs a comment on what it does.

Reply:
Added description for all kerberos related fields